api/Migrations/20220328124415_Attendance.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace YonderfulApi.Migrations
+{
+    public partial class Attendance : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "Autocancel",
+                table: "Events",
+                newName: "AutoCancel");
+
+            migrationBuilder.CreateTable(
+                name: "Attendace",
```

Comment:
You have a typo in the name of the table; should be Attendance

Reply:
It is renamed in another migration